line-colors.csv

```diff
@@ -993,6 +993,16 @@ vbn-nwb-regio-s-bahn,RS 3,nordwestbahn,4-n1-rs3,#93bf30,#ffffff,,rectangle-round
 vbn-nwb-regio-s-bahn,RS 30,nordwestbahn,4-n1-rs30,#4faf3b,#ffffff,,rectangle-rounded-corner
 vbn-nwb-regio-s-bahn,RS 4,nordwestbahn,4-n1-rs4,#e2000b,#ffffff,,rectangle-rounded-corner
 vbn-nwb-regio-s-bahn,RS 6,nordwestbahn,4-n1-rs6,#899bb9,#ffffff,,rectangle-rounded-corner
+vbn-bsag,1,,8-webbtr-1,#00a54f,#ffffff,,rectangle
+vbn-bsag,1S,,8-webbtr-1S,#00a54f,#ffffff,,rectangle
```

Comment:
HAFAS line code needs to be en all lower case!

Reply:
my bad, didn't know that!!!